packages/cardano-services/src/Http/HttpServer.ts

```diff
@@ -180,4 +156,69 @@ export class HttpServer extends RunnableModule {
 
     return serverClosePromise(this.server);
   }
+
+  // eslint-disable-next-line sonarjs/cognitive-complexity
+  private async initMetrics() {
+    const promRegistry = new Registry();
+    const anyService = this.#dependencies.services[0];
```

Comment:
Question: doesn't matter which service we pick here as first, correct? @iadmytro

Reply:
yes, since services reporting same node sync status